notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -61,30 +63,27 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial demonstrates how to use Vertex AI Pipelines to rapid prototype a model using both AutoML and BQML, do an evaluation comparison, for a baseline, before progressing to a custom model.\n",
-        "\n",
-        "\n",
-        "<img src=\"https://storage.googleapis.com/rafacarv-public-bucket-do-not-delete/abalone/automl_and_bqml.png\" />\n",
+        "This tutorial demonstrates how to use Vertex AI Pipelines to rapidly prototype a model using both AutoML and BQML, evaluate and compare them for a baseline model before progressing to a custom model.\n",
         "\n",
         "Learn more about [AutoML components](https://cloud.google.com/vertex-ai/docs/pipelines/vertex-automl-component) and [BigQuery ML components](https://cloud.google.com/vertex-ai/docs/pipelines/bigqueryml-component)."
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "k-82uiXlTjvw"
+        "id": "c6fc06d85572"
       },
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use `Vertex AI Predictions` for rapid prototyping a model.\n",
+        "In this tutorial, you learn how to use Vertex AI pipelines for rapid prototyping a model.\n",
```

Comment:
- change: Vertex AI pipelines
    - to: Vertex AI Pipelines (that's the product name)